lib/graphql/fragment_cache/fragment.rb

```diff
@@ -12,22 +12,22 @@ class Fragment
 
       class << self
         def read_multi(fragments)
-          unless FragmentCache.cache_store.respond_to?(:read_multi)
-            return fragments.map { |f| [f, f.read] }.to_h
-          end
+          return fragments.map { |f| [f, f.read] }.to_h unless FragmentCache.cache_store.respond_to?(:read_multi)
```

Comment:
This is just a formatting change, right?

Reply:
Yes, it still functions exactly the same, the change is just from a later version of Rubocop's auto-formatting rules. I have format-on-save turned on in my text editor. I'll revert those changes so that only the code I changed and tested is modified.